jest.config.js

```diff
@@ -11,11 +11,24 @@ module.exports = {
   ],
   projects: [
     {
-      displayName: "test",
+      displayName: "test-node",
       setupFiles: ["<rootDir>/tests_config/run_spec.js"],
       testRegex: "jsfmt\\.spec\\.js$|__tests__/.*\\.js$",
       snapshotSerializers: ["jest-snapshot-serializer-raw"],
-      testEnvironment: "node"
+      testEnvironment: "node",
+      globals: {
+        STANDALONE: false
+      }
+    },
+    {
+      displayName: "test-standalone",
+      setupFiles: ["<rootDir>/tests_config/run_spec.js"],
+      testRegex: "jsfmt\\.spec\\.js$|__tests__/.*\\.js$",
+      snapshotSerializers: ["jest-snapshot-serializer-raw"],
+      testEnvironment: "jsdom",
+      globals: {
+        STANDALONE: true
+      }
```

Comment:
Maybe we can add SKIP_STANDALONE_TEST env for better test experience?

Reply:
Sorry, I'm not sure how your question relates to the code snippet?

Reply:
@Loilo i mean `SKIP_STANDALONE_TEST=true npm run test/yarn test` (and no test for bundled)